python/Ganga/Core/GangaRepository/SessionLock.py

```diff
@@ -443,7 +422,8 @@ def shutdown(self):
         self.locked = set()
         try:
             global session_lock_refresher
-            session_lock_refresher.stop()
+            if session_lock_refresher and not session_lock_refresher.should_stop():
+                session_lock_refresher.stop()
```

Comment:
No idea, but from my understanding there is no need to even if the function is safe, so why do it?